package.json

```diff
@@ -33,6 +34,9 @@
   "scripts": {
     "test": "mocha"
   },
+  "dependencies": {
+    "url": "^0.11.4"
```

Comment:
this alone doesn’t do anything - requiring “url” always gets the core module, no matter what’s installed.